.phpcs.xml.dist

```diff
@@ -46,7 +46,21 @@
 
 	<arg name="colors"/>
 
+	<!-- Ignore external libraries -->
 	<exclude-pattern>/node_modules/*</exclude-pattern>
 	<exclude-pattern>/vendor/*</exclude-pattern>
+
+	<!-- Ignore WordPress and such checked out for Docker -->
+	<exclude-pattern>/docker/mu-plugins/0-sandbox.php</exclude-pattern>
+	<exclude-pattern>/docker/wordpress-develop/*</exclude-pattern>
+	<exclude-pattern>/docker/wordpress/*</exclude-pattern>
+
+	<!-- Ignore WordPress included in packages -->
 	<exclude-pattern>/packages/*/wordpress/*</exclude-pattern>
+
+	<!-- Ignore built files -->
+	<exclude-pattern>/_inc/blocks/*</exclude-pattern>
+	<exclude-pattern>/_inc/build/*</exclude-pattern>
+	<exclude-pattern>/_inc/jetpack-strings.php</exclude-pattern>
```

Comment:
We should be able to remove this one now; we don't use that file anymore.

Reply:
Done, and removed it from .gitignore too.